### DISCUSSION TITLE
是否支持内网部署

### DISCUSSION THREAD

**Discussion by zzywbo** [opentiny/tiny-engine#243] [Q&A]
是否支持内网部署？能出一个部署教程的视频吗？感谢！！！！

**Comment by zzywbo** [comment]
在本机启动成功了，请问官网中的“PDM 元数据审批电子流”项目有源码可以分享下吗？

**Reply by kedenLi** [reply]
你好，可以详细说一下你的项目背景和诉求吗？或者你可以通过我邮箱，我们进一步沟通！
